.github/workflows/Tests.yaml

```diff
@@ -329,6 +330,9 @@ jobs:
         working-directory: /work/build
         run: |
           make -j2 RunTests
+          # Find and delete object files in the build dir for unit tests.
+          # This reduces the size of the build dir by at least 1.5GB
```

Comment:
it's probably fine to have these comments in the `run` script, but keep in mind they are actually lines in the shell script. If you move the comment above line 331 then it's a YAML comment instead, which feels more comfortable imo

Reply:
It runs fine on my fork. I'd rather have the comments be right where the code it is commenting on is, otherwise the comment is already lost from the code it's describing